server/controllers/isAuth.js

```diff
@@ -0,0 +1,19 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = (req, res, next) => {
+  const {
+    cookies: { token },
+  } = req;
+
+  if (token) {
+    jwt.verify(token, process.env.SECRET_KEY, (err, userToken) => {
+      if (err) {
+        return next(err);
+      }
+      req.dataUser = userToken;
+      return next();
+    });
+  } else {
+    res.redirect('/login');
```

Comment:
We'll redirect from  the client-side, all we have to do here is to send `json` with a descriptive message and a `401` statusCode